src/pybind/mgr/dashboard/frontend/src/app/shared/components/deletion-modal/deletion-modal.component.html

```diff
@@ -13,39 +13,23 @@
         <ng-container *ngTemplateOutlet="description"></ng-container>
         <p>
           <ng-container i18n>
-            To confirm the deletion, enter
+            Are you sure you want to delete the selected
           </ng-container>
-          <kbd>{{ pattern }}</kbd>
-          <ng-container i18n>
-            and click on
-          </ng-container>
-          <kbd>
-            <ng-container *ngTemplateOutlet="deletionHeading"></ng-container>
-          </kbd>.
+          {{ metaType }}?
         </p>
         <div class="form-group"
              [ngClass]="{'has-error': deletionForm.showError('confirmation', formDir)}">
-          <input type="text"
-                 class="form-control"
-                 name="confirmation"
-                 id="confirmation"
-                 [placeholder]="pattern"
-                 [pattern]="escapeRegExp(pattern)"
-                 autocomplete="off"
-                 (keyup)="updateConfirmation($event)"
-                 formControlName="confirmation"
-                 autofocus>
-          <span class="help-block"
-                *ngIf="deletionForm.showError('confirmation', formDir, 'required')"
-                i18n>
-          This field is required.
-        </span>
-          <span class="help-block"
-                *ngIf="deletionForm.showError('confirmation', formDir, 'pattern')">
-          '{{ confirmation.value }}'
-          <span i18n>doesn't match</span>
-          '{{ pattern }}'.
-        </span>
+          <div class="checkbox checkbox-primary">
+            <input type="checkbox"
+                   name="confirmation"
+                   id="confirmation"
+                   formControlName="confirmation"
+                   autofocus>
+            <label i18n
+                   for="confirmation">
+              To confirm the deletion, please check the checkbox.
```

Comment:
IMO, "checkbox" is a technical term that we should avoid to use in the UI.

I'd suggest somethink like "`I'm sure I want to proceed with the deletion`" (or equivalent).